mmv1/products/compute/api.yaml

```diff
@@ -13397,6 +13397,13 @@ objects:
               - :ERRORS_ONLY
               - :TRANSLATIONS_ONLY
               - :ALL
+      - !ruby/object:Api::Type::Boolean
+        name: enableDynamicPortAllocation
+        description: |
+          Using dynamic port allocation lets the NAT gateway allocate different numbers of ports to each VM based 
+          on usage. For more information see the [official documentation](https://cloud.google.com/nat/docs/overview#specs-rfcs).
```

Comment:
We probably want to match the description with API documentation https://cloud.google.com/compute/docs/reference/rest/v1/routers